acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,172 @@
+            if loaded_count % 10 == 0:
+                logger.info(
+                    f"Processed {loaded_count}/{total_count} fingerprints ({loaded_count / total_count * 100:.2f}%)"
+                )
+
+    logger.info(f"Initial data load complete. Processed {loaded_count} fingerprints.")
+
+
+async def replicate_from_pg() -> None:
+    async with AsyncExitStack() as exit_stack:
+        pgc = await asyncpg.connect(
+            "postgresql://acoustid:acoustid@localhost:5432/acoustid_fingerprint"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)